mpt_extension_sdk/runtime/djapp/conf/default.py

```diff
@@ -34,22 +34,22 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ("*")
+ALLOWED_HOSTS = ["*"]
```

Comment:
Amm.. it is recommend to have here immutable types, so I would insist to have tuples here, also the errors says

```
django.core.exceptions.ImproperlyConfigured: The ALLOWED_HOSTS setting must be a list or a tuple.
```

I think the reason is that `,` is missed here

```
ALLOWED_HOSTS = ("*",)
```